gulpfile.js

```diff
@@ -507,6 +513,46 @@ function release_deb(arch) {
     }));
 }
 
+function release_rpm(arch, done) {
+
+    // Check if dpkg-deb exists
+    if (!commandExistsSync('rpmbuild')) {
+        console.warn('rpmbuild command not found, not generating rpm package for ' + arch);
+        return done();
+    }
+
+    // The buildRpm does not generate the folder correctly, manually
+    createDirIfNotExists(RELEASE_DIR);
+
+    var options = {
+             name: pkg.name,
+             version: pkg.version,
+             buildArch: getLinuxPackageArch('rpm', arch),
```

Comment:
`getLinuxPackageArch()` needs to be defined (grab it from `gulpfile.js`  of the betaflight-configurator.

Reply:
Oh, that one slipped through.